src/python/doc/persistence_graphical_tools_user.rst

```diff
@@ -67,10 +67,17 @@ If you want more information on a specific dimension, for instance:
 
     import matplotlib.pyplot as plot
     import gudhi
-
     # rips_on_tore3D_1307.pers obtained from write_persistence_diagram method
     persistence_file=gudhi.__root_source_dir__ + \
         '/data/persistence_diagram/rips_on_tore3D_1307.pers'
-    gudhi.plot_persistence_density(persistence_file=persistence_file,
-        max_intervals=0, dimension=1, legend=True)
+    birth_death = gudhi.read_persistence_intervals_in_dimension(
+        persistence_file=persistence_file,
+        only_this_dim=1)
+    pers_diag = [(1, (elt[0], elt[1])) for elt in birth_death]
```

Comment:
Does `(1, elt)` work as well as `(1, (elt[0], elt[1]))`?
We really need to fix #27 some day...